src/main/resources/templates/register.html

```diff
@@ -32,7 +32,7 @@ <h3 class="ui dividing header" th:text="#{register.userdata}">Nutzerdaten</h3>
 
         <div class="field">
             <label for="name">Name</label>
-            <input id="name" name="name" th:field="*{name}" th:errorclass="fieldError" type="text" required="required"/><br/>
```

Comment:
I wouldn't actually want to remove those attributes as in general, browser validation is a good thing to have.

Reply:
Good point. I only thought about the @NotEmpty message. But if you add other validations, they should be shown. And if your browser does not support the required attribut there is the @NotEmpty fallback in Spring.

Reply:
I inserted it again.